### PR TITLE
Comprehensive research on AI ML Applications

### PR DESCRIPTION
**Pull request by MisaAghera**
Added review on comprehensive research on AI ML applications.

### CODE REVIEW

**Comment by MisaAghera** [comment]
[Conduct comprehensive research on AI/ML Applications](https://trello.com/c/yxpMGphH/15-conduct-comprehensive-research-on-ai-ml-applications)

**Comment by MisaAghera** [comment]
[Conduct comprehensive research on AI/ML Applications](https://trello.com/c/yxpMGphH/15-conduct-comprehensive-research-on-ai-ml-applications)

File: Documentation/Comprehensive research on AI ML Applications.pdf
Comment (KeerthiKutumbaka) [inline Documentation/Comprehensive research on AI ML Applications.pdf:1]: Hi @MisaAghera , Could you please recheck the sections and upload it again? 

Thank you.

Comment (MisaAghera) [reply Documentation/Comprehensive research on AI ML Applications.pdf:1]: acknowledged and updated

**Review by KeerthiKutumbaka — APPROVED** [review]

**Pull request merged**